modules/ldsg.js

```diff
@@ -5,6 +5,40 @@ const Augur = require("augurbot-ts"),
   /** @type {string[]} */
   banned = require("../data/banned.json").features.suggestions;
 
+const hasLink = /http(s)?:\/\/(\w+(-\w+)*\.)+\w+/;
+const affiliateLinks = {
+// amazon: { //Functionality can be renabled if amazon will let us get a affiliate
+  //  site: "Amazon",
+  //  affiliate: "Amazon Affiliate",
+  //  test: /amazon\.(com|co\.uk)\/(\w+(-\w+)*\/)?(gp\/product|dp)\/(\w+)/i,
+  //  tag: /tag=ldsgamers-20/,
+  //  link: (match) => `https://www.${match[0]}?tag=ldsgamers-20`
+  //  },
+  cdkeys: {
+    site: "CDKeys.com",
+    affiliate: "CDKeys Affiliate",
+    test: /cdkeys\.com(\/\w+(-\w+)*)*/i,
+    tag: /mw_aref=LDSGamers/i,
+    link: match => `https://www.${match[0]}?mw_aref=LDSGamers`
```

Comment:
Same thing as the next comment, just with a different type. Our ESLint configuration is currently a little broken, so we have to ignore an error message. The disabling comment will be removed soon but is needed at this point to pass the test
```suggestion
    // eslint-disable-next-line jsdoc/no-undefined-types
    /** @param {RegExpExecArray} match */
    link: match => `https://www.${match[0]}?mw_aref=LDSGamers`
```